iphone/Maps/Classes/CarPlay/CarPlayService.swift

```diff
@@ -37,13 +37,19 @@ final class CarPlayService: NSObject {
     self.window = window
     self.interfaceController = interfaceController
     self.interfaceController?.delegate = self
-    sessionConfiguration = CPSessionConfiguration(delegate: self)
+    let configuration = CPSessionConfiguration(delegate: self)
```

Comment:
Is this intermediate variable really necessary?

Reply:
Nope. But it is clearer to use a switch-case for the enum `contentStyle` and avoid if-elses... 

Probably this will be more clear:
```swift
sessionConfiguration = CPSessionConfiguration(delegate: self)
    // Try to use the CarPlay unit's interface style.
    if #available(iOS 13.0, *), let contentStyle = sessionConfiguration?.contentStyle  {
      switch contentStyle {
      case .light:

```

Reply:
I see, these optionals sometimes make development more complex :) LGTM